recipes/reproject/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.3" %}
+
+package:
+  name: reproject
+  version: {{version}}
+
+source:
+  fn: reproject-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/r/reproject/reproject-{{ version }}.tar.gz
+  md5: 6e2b2f7f178459da3e56405a4ad832d6
+
+build:
+  number: 0
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
+
+  run:
+    - python
+    - numpy x.x
```

Comment:
See my comment below. (I missed the `astropy_helpers` `submodule` in my first pass :smile:)